Dockerfile

```diff
@@ -0,0 +1,44 @@
+### 1. Get Linux
+FROM alpine:3.7
+
+ADD gpg /root/gpg
```

Comment:
This cause the build to fail if the `gpg` dir is missing. Also, this isn't needed as you have mounted in your run example